client/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+
+[package]
+edition = "2021"
+name    = "svc-pricing-client"
+version = "0.1.0"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+prost       = "0.11.0"
+prost-types = "0.11.1"
+tonic       = "0.8.1"
+
+[dev-dependencies]
+tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
+
+[lib]
+name = "cargo_client"
```

Comment:
svc-pricing-client? "copy pasta"

You can also use this in the example instead of re-declaring the grpc mod

Reply:
What do you mean? Do you want me to rename the lib to "svc_pricing_client"? the kebab case wouldn't work for lib names.

Reply:
svc_pricing_client instead of cargo_client

Reply:
[5dcf464](https://github.com/Arrow-air/svc-pricing/pull/1/commits/5dcf464afe784dbbd43a522f1edd1e56726d2c12)